src/handlers/site_upload/api_gateway_authorizer.py

```diff
@@ -28,7 +27,7 @@ def lambda_handler(event, context):
         if auth_token not in user_db.keys() or auth_header[0] != "Basic":
             raise AuthError
     except (AuthError, KeyError):
-        raise AuthError(event)  # pylint: disable=raise-missing-from
+        raise AuthError(event)  # noqa: B904
```

Comment:
because this is in the 'this code came directly from AWS and we're trying to leave that as much as found as possible' bin? you could talk me out of it.